torch/nn/modules/module.py

```diff
@@ -296,17 +296,20 @@ def to(self, *args, **kwargs):
 
         This can be called as
 
-        .. function:: to(device)
-
         .. function:: to(dtype)
 
-        .. function:: to(device, dtype)
+        .. function:: to(device, dtype=None, non_blocking=True)
```

Comment:
Should this match `Tensor.to` which defaults to `non_blocking=False`?

Reply:
Good catch!